designs/virtualization/4.5/Scheduling/Node-selector/node-selector.md

```diff
@@ -0,0 +1,37 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Node Selector
+
+Introducing Node-selector as it appears in-page and the Edit modal.
+
+![Node selector field in the Scheduling section ](img/Node-selector-0-0.jpg)
+The node-selector field is pulled from the Workload details section and placed under the Scheduling section.
+
+In this example there is no node-selector set, meaning, no node labels added.
+Clicking on the edit (pencil) button will open the Edit modal.
+Clicking on the "O labels" link text will open that modal as well.
+
+
+![Node selector modal ](img/Node-selector-1-0.jpg)
+The modal consists of:
+- key=value pair text fields followed by a Remove button
+- Button to add another pair, or label
+- External link to the nodes list
+- "Nodes Checker", active notification that provides feedback if nodes found for the labels above
+
+When there is only one row of key=value pair, the Remove action will be disabled
+
+![Node selector modal ](img/Node-selector-1-1.jpg)
+On typing some value within any of the fields, the Node Checker will check for nodes with matching labels, with a time out of [1] second to kick in.
+
+![Node selector modal ](img/Node-selector-1-2.jpg)
+In case a match found, The Node Checker will fire a positive feedback.
+It will also include a link to view the matching nodes within a popover, on hover or on click.
```

Comment:
```suggestion
It will also include a link to view the matching nodes within a popover on click.
```

I believe popovers like this are always on click, at least out of the box in PatternFly. Dark-colored [tooltips](https://www.patternfly.org/v4/documentation/react/components/tooltip) appear on hover, but we should try to use those rarely.